agile-development/stand-ups/social-question/readme.md

```diff
@@ -0,0 +1,39 @@
+# Social question of the day
+
+The social question of the day is an optional short question to follow the [three project questions](../readme.md) in the daily stand-up. It develops team cohesion and interpersonal trust over the course of an engagement by facilitating the sharing of chosen personal context.
+
+If the team decides to incorporate a social question-of-the-day into the stand-up, the questions can be selected by the facilitator or sourced from the attendees. The latter is faster and the former will likely result in more diverse questions.
```

Comment:
It would be best to suggest that the facilitator should select the question ahead of time as it makes it more efficient. I know we have not been doing this recently but we are in a small group at the moment. 
I would also change this guidance in the main social question section above.

Reply:
Agreed that the question should be selected ahead of time. I changed it to suggest either independent selection by the facilitator or selection from some async suggestions. What do you think?

Reply:
perfect!